content/zh/docs/devops-user-guide/examples/multi-cluster-project-example.md

```diff
@@ -77,9 +74,9 @@ weight: 11420
        DOCKERHUB_USERNAME = 'Your Docker Hub username'
        APP_NAME = 'devops-go-sample'
        // ‘dockerhub-go’ 即您在 KubeSphere 控制台上创建的 Docker Hub 凭证 ID
```

Comment:
I think `dockerhub-go` here should be updated to `dockerhub` too?

Reply:
of course